src/store/reducers/tenant/tenant.ts

```diff
@@ -24,7 +24,7 @@ const SET_METRICS_TAB = 'tenant/SET_METRICS_TAB';
 const CLEAR_TENANT = 'tenant/CLEAR_TENANT';
 const SET_DATA_WAS_NOT_LOADED = 'tenant/SET_DATA_WAS_NOT_LOADED';
 
-const initialState = {loading: false, wasLoaded: false};
+const initialState = {loading: true, wasLoaded: false};
```

Comment:
Tenant diagnostics tab content was requested twice, because requests were sent before state was set as loading and after tenant data is fully loaded. This fix seems to be the easiest, tenant content considered loading from the start, there is no attempt to load tab content